src/modules/flow/flow.c

```diff
@@ -404,16 +407,16 @@ uint8_t compute_flow(uint8_t *image1, uint8_t *image2, float x_rate, float y_rat
 	const uint16_t hist_size = 2*(winmax-winmin+1)+1;
 
 	/* variables */
-        uint16_t pixLo = SEARCH_SIZE + 1;
-        uint16_t pixHi = FRAME_SIZE - (SEARCH_SIZE + 1) - TILE_SIZE;
-        uint16_t pixStep = (pixHi - pixLo) / NUM_BLOCKS + 1;
+    uint16_t pixLo = SEARCH_SIZE + 1;
+    uint16_t pixHi = FRAME_SIZE - (SEARCH_SIZE + 1) - TILE_SIZE;
+    uint16_t pixStep = (pixHi - pixLo) / NUM_BLOCKS + 1;
```

Comment:
Looks like this repository wants tabs :sob:

Reply:
Tabs - done